test/utils.py

```diff
@@ -10,3 +12,100 @@ def get_autzen_file():
 
 def get_data_dir():
     return DATADIRECTORY
+
+
+def generate_test_file():
```

Comment:
What does this do? Generate a file with random points within the bounds below? Might want to add a docstring